py/newsletter.py

```diff
@@ -24,13 +25,19 @@
 # create the document body
 with doc:
     # enclose all of the elements within div elements (currently set to ef-styling)
-    with div(cls='wrapper').add(div(cls='container')).add(table(cls='table outer')):
-        _head()
-        _greetings(greet_title)
-        _bodylineleft()
-        _bodylineright()
-        _bodynoline()
-        _foot()
+    with div(cls='sk-wrapper').add(div(cls='sk-webkit')).add(table(cls='sk-table sk-outer sk-height')):
```

Comment:
are the `sk-*` classnames still relevant? or can we switch to all being on one css file with out the prefixes

Reply:
Yeah i can make those changes

Reply:
I removed the sk's for the classes in the main py file for now. I'll gradually remove the sk's for the other classes as I use them in the different python files for each section. This is so I can gradually test and make sure changing every class name doesn't affect the display.